help/develop/domains/chem/cheminformatics.md

```diff
@@ -31,59 +31,36 @@ descriptors(table, column, descriptors);
 ```
 
 The three first functions, `searchSubstructure`, `getSimilarities` and `findSimilar`, are currently made client-based,
```

Comment:
This sentence should be rewritten. For instance:
`searchSubstructure, getSimilarities and findSimilar functions are client-based and use RDKit JS library`.

Reply:
Done